.github/workflows/api.yml

```diff
@@ -50,7 +50,15 @@ jobs:
           chmod +x $DOCKER_CONFIG/cli-plugins/docker-compose
           docker compose version
       - name: Build containers
-        run: cd api && make build
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::613752250312:role/github_production_deploy
+          aws-region: us-east-2
+        run: cd api && source set_envs.sh && make build
       - name: Run tests
-        run: cd api && make api-test
+        uses: aws-actions/configure-aws-credentials@v4
+        with:
+          role-to-assume: arn:aws:iam::613752250312:role/github_production_deploy
+          aws-region: us-east-2
+        run: cd api && source set_envs.sh && make api-test
```

Comment:
Are these changes needed?

Reply:
Only if we want to test with active API keys to see that functionality for ie ACLED, STAC, etc works.  For now I've removed since it requires modifying permissions and seems like we weren't doing that before.